pom.xml

```diff
@@ -1128,6 +1171,46 @@
       <properties>
         <java.version>17</java.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.11.0</version>
```

Comment:
Extract as a maven property too.

Reply:
Addressed as part of https://github.com/apache/incubator-celeborn/pull/2025#discussion_r1369491406